harness/arrayContains.js

```diff
@@ -1,10 +1,16 @@
-//-----------------------------------------------------------------------------
-function arrayContains(arr, expected) {
+/**
+ * Verify that a subArray is contained within an array.
+ *
+ * @param {Array} array
+ * @param {Array} subArray
+ */
+
+function arrayContains(array, subArray) {
   var found;
-  for (var i = 0; i < expected.length; i++) {
+  for (var i = 0; i < subArray.length; i++) {
     found = false;
-    for (var j = 0; j < arr.length; j++) {
-      if (expected[i] === arr[j]) {
+    for (var j = 0; j < array.length; j++) {
+      if (subArray[i] === array[j]) {
         found = true;
         break;
       }
```

Comment:
The only real change here is that I've added a comment to the top, which documents the expected behavior and arguments.